Resources/Prototypes/_LostParadise/Maps/Hell/H_asterisk.yml

```
@@ -0,0 +1,77 @@
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_Kaeri.yml
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Станция Астериск {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+          #service
```

yamllint

[warning] missing starting space in comment

(comments)